From 692945ced4a7d7375e1b92c32200b8de936bbc57 Mon Sep 17 00:00:00 2001
From: John Audia <therealgraysky@proton.me>
Date: Sat, 26 Aug 2023 08:36:37 -0400
Subject: [PATCH] kernel: bump 6.1 to 6.1.48

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.48

No patches changed in this bump, only update was to checksum.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne, filogic/xiaomi_redmi-router-ax6000-ubootmod
Run-tested: x86/64/AMD Cezanne, filogic/xiaomi_redmi-router-ax6000-ubootmod

Signed-off-by: John Audia <therealgraysky@proton.me>
---
 include/kernel-6.1 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/kernel-6.1 b/include/kernel-6.1
index 61bad0e5ca7cf..599ad1461eead 100644
--- a/include/kernel-6.1
+++ b/include/kernel-6.1
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.1 = .47
-LINUX_KERNEL_HASH-6.1.47 = 93d58b6af007a5f44dd26831ff310707deb1ab9380c5136a534287eb3fddfcab
+LINUX_VERSION-6.1 = .48
+LINUX_KERNEL_HASH-6.1.48 = c606cbd0353e677df6fae73cc16ba3c9244b98372ed7771d551024016f55ac31
